### ISSUE TITLE
user can't upload files in batches

### ISSUE THREAD

**Issue by ghost** [Nerodesk/nerodesk#151]
Current version of README doesn't say anything about batch file upload feature. This is a bug, since this feature is important and will improve usability of the system. Let's modify the README and document this feature. Then, let's implement it and test.

**Comment by ghost** [comment]
also see [#94](https://github.com/teamed/nerodesk/issues/94)

**Comment by yegor256** [comment]
@k0r take a look at how I modified the title of the ticket and its description

**Comment by k0r** [comment]
@yegor256 thanks.

**Comment by yegor256** [comment]
@karato it's a bug, let's dispatch it

**Comment by karato** [comment]
> @karato it's a bug, let's dispatch it

@yegor256 right, I added the `bug` tag

**Comment by karato** [comment]
@ggajos do this task

**Comment by ggajos** [comment]
@karato please assign someone else, thank you

**Comment by karato** [comment]
> @karato please assign someone else, thank you

@ggajos right, I'll try to find someone else for this task

**Comment by karato** [comment]
@pinaf do this task

**Comment by pinaf** [comment]
@karato please see PR #184

**Comment by pinaf** [comment]
@k0r PR merged. please check and close if alright :)

**Comment by k0r** [comment]
@pinaf current release tag 0.3.1 seems to only support single file uploads.

**Comment by pinaf** [comment]
@k0r of course, the functionality can't be completed in a single issue. the puzzle I added will allow the feature to continue.

**Comment by k0r** [comment]
@pinaf lets keep the discussion going until the implementation is live.

**Comment by pinaf** [comment]
@k0r hmm not sure that's how PDD works...

**Comment by k0r** [comment]
@pinaf let's see what @yegor256 says -- usually I keep the ticket open until the implementation is accessible.

**Comment by k0r** [comment]
@pinaf thanks ;-]

**Comment by yegor256** [comment]
@pinaf it's a valid concern. each ticket should give some value to the user. it may not work as expected, but the user should get some increment in the functionality. in this puzzle you wrote some code that make sense to you, but has no value to the user. see http://www.yegor256.com/2015/02/12/top-down-design.html

**Comment by pinaf** [comment]
@k0r @yegor256 yeah I went bottom-up instead of top-down, sorry :(
 I guess I should have started from the upload page!

**Comment by yegor256** [comment]
@pinaf exactly

**Comment by pinaf** [comment]
@yegor256 it is a bad habit - has to do with always thinking about the underlying structure first :) I'll keep it in mind next time.

**Comment by yegor256** [comment]
@pinaf yes, it's a matter of habit. We all have that. We like to code for ourselves first. We like programmers and hate users :) Try to change this mindset and start with satisfying a user first and then thinking about design/code/programming.

**Comment by karato** [comment]
once `151-6767899c` puzzle is resolved (later, in another ticket), this ticket will be fully complete

**Comment by karato** [comment]
@pinaf 7 days already, in 48 hours I will re-assign. -30 added to your rating, current score is: [+6388](http://www.netbout.com/b/35128?open=rating)

**Comment by pinaf** [comment]
@yegor256 @karato just took points away from me but this was merged 6 days ago..

**Comment by pinaf** [comment]
@karato we need more time here

**Comment by karato** [comment]
@pinaf the project is currently on pause, please stop working with this task until further notice

**Comment by karato** [comment]
> @karato we need more time here

@pinaf sure, thanks for telling me, take your time

**Comment by pinaf** [comment]
@yegor256 any chance I get these points back?

**Comment by karato** [comment]
@pinaf the project is back on track, please continue

**Comment by pinaf** [comment]
@karato we need more time here

**Comment by pinaf** [comment]
@yegor256 can we close this one?

**Comment by karato** [comment]
> @karato we need more time here

@pinaf take your time, I am aware that we're waiting here

**Comment by yegor256** [comment]
@pinaf well, as I see above, the task is not done. you created some changes in the code, but for the user perspective nothing was changed. like I explained above, we should go from top to bottom, not the other way around

**Comment by pinaf** [comment]
@yegor256 I agree with that. my question was motivated because of the recent changes in the project

**Comment by pinaf** [comment]
@karato we're waiting for #198 here

**Comment by karato** [comment]
> @karato we're waiting for #198 here

@pinaf agreed, we'll wait for #198

**Comment by yegor256** [comment]
@karato pls assign someone else here

**Comment by karato** [comment]
> @karato pls assign someone else here

@yegor256 I deducted 30 points from your rating

**Comment by karato** [comment]
> @karato pls assign someone else here

@yegor256 got it, someone else will be assigned soon

**Comment by karato** [comment]
@maurezen this is your task

**Comment by yegor256** [comment]
@karato it's postponed

**Comment by karato** [comment]
> @karato it's postponed

@yegor256 sure, thanks, I added "postponed" tag to it

**Comment by karato** [comment]
> @karato it's postponed

@yegor256 no problem, I will try to find somebody else
